src/Microsoft.IdentityModel.Clients.ActiveDirectory/Internal/Flows/AcquireTokenHandlerBase.cs

```diff
@@ -301,20 +319,20 @@ protected virtual async Task PreRunAsync()
         {
             return Task.FromResult(false);
         }
-        
+
         protected async Task UpdateAuthorityAsync(string updatedAuthority)
         {
-            if(!Authenticator.Authority.Equals(updatedAuthority, StringComparison.OrdinalIgnoreCase))
+            if (!Authenticator.Authority.Equals(updatedAuthority, StringComparison.OrdinalIgnoreCase))
             {
-                await Authenticator.UpdateAuthorityAsync(updatedAuthority, RequestContext).ConfigureAwait(false);
+                await Authenticator.UpdateAuthorityAsync(updatedAuthority, RequestContext, ServiceBundle).ConfigureAwait(false);
```

Comment:
>ServiceBundle [](start = 91, length = 13)

nit: in other places we have service bundle first

Reply:
yep. thanks. we have it first in MSAL, so was going for consistency. missed this one.

---
In reply to: [267510352](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1530#discussion_r267510352) [](ancestors = 267510352)